components/Calculator.js

```diff
@@ -0,0 +1,93 @@
+// class Field extends React.Components {
```

Comment:
**Changes Required**
- [ ] Let us try not to commit commented out code to our repo. It makes it look unprofessional. You can use `git history` to track your changelog. Let us try to remove these and delete this directory `src/components` as a whole.